apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.types.ts

```diff
@@ -2,28 +2,33 @@ import { z } from 'zod';
 
 import { AIGradingStatsSchema } from '../../../ee/lib/ai-grading/types.js';
 import {
-  AssessmentQuestionSchema,
-  IdSchema,
-  type InstanceQuestionGroup,
-  InstanceQuestionSchema,
-} from '../../../lib/db-types.js';
+  RawStaffInstanceQuestionSchema,
+  StaffAssessmentQuestionSchema,
+  type StaffInstanceQuestionGroup,
+  StaffInstanceQuestionSchema,
+} from '../../../lib/client/safe-db-types.js';
+import { IdSchema } from '../../../lib/db-types.js';
 import type { RubricData } from '../../../lib/manualGrading.types.js';
 
-export const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
+export const InstanceQuestionRowSchema = z.object({
+  instance_question: StaffInstanceQuestionSchema,
   assessment_open: z.boolean(),
   uid: z.string().nullable(),
   assigned_grader_name: z.string().nullable(),
   last_grader_name: z.string().nullable(),
-  assessment_question: AssessmentQuestionSchema,
+  assessment_question: StaffAssessmentQuestionSchema,
```

Comment:
👀 I know this is unchanged, but it seems very wasteful to fetch the exact same assessment question over and over again. Surely we get the assessment question from the page as a whole?